utils/misc/misc.go

```diff
@@ -1426,3 +1426,10 @@ func CopyStringMap(originalMap map[string]string) map[string]string {
 	}
 	return newMap
 }
+
+func GetRouterIdentifier(id, name string) string {
```

Comment:
it would help to transform this into a `DestinationT` receiver function, e.g.

```
func (d DestinationT) GetRouterIdentifier() string {
	if config.GetBool(fmt.Sprintf("Router.%s.isolateDestID", d.DestinationDefinition.Name), false) {
		return d.ID
	}
	return d.DestinationDefinition.Name
}
```

Reply:
Do we really need this in misc? I wouldn't want to import backend-config into misc , so should I move this to backend-config file?

Reply:
of course this would belong inside backend-config :)